schema/show.js

```diff
@@ -70,7 +70,8 @@ const ShowType = new GraphQLObjectType({
     name: {
       type: GraphQLString,
       description: 'The exhibition title',
-      resolve: ({ name }) => name.trim(),
+      resolve: ({ name }) =>
+        isExisty(name) ? name.trim() : '',
```

Comment:
Seeing as the schema indicates that this field may be `null` (i.e. it’s not designated not-null), clients should already be aware of that and use the `null` sentinel as a way to determine what to do. I think `null` is better than an empty string for those cases.